.github/workflows/testNginxForAzureDeployV1.yml

```diff
@@ -0,0 +1,69 @@
+# File: .github/workflows/testNginxForAzureDeployV1.yml
+
+name: Test Github action to update NGINX for Azure configurations and certificates
+on:
+  - push
+  - workflow_dispatch
+
+env:
+  NGINX_DEPLOYMENT_NAME: github-action-test-deployment
```

Comment:
Can we make a separate test workflow since you have your own long lived setup? We just want to make sure the tests we own are able to be managed by our test team and we will not have visibility into your test setup.

Might be worthwhile to setup a very short meeting to discuss this.

Reply:
As per our discussion, I have removed the test workflow